cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsSetup.java

```diff
@@ -128,7 +128,8 @@ public void prerequisites(AuthenticatedContext ac, CloudStack stack, Persistence
     private void validateRegionAndZone(CloudCredential cloudCredential, Location location) {
         CloudRegions regions = awsPlatformResources.regions(cloudCredential, location.getRegion(), Collections.emptyMap());
         List<AvailabilityZone> availabilityZones = regions.getCloudRegions().get(location.getRegion());
-        if (!availabilityZones.contains(location.getAvailabilityZone())) {
+        if (location.getAvailabilityZone() != null
```

Comment:
is it possible that the az is null here?

Reply:
The FreeIPA cloud formation template was simplified. The AZs were needed when creating subnets which only cloudbreak core does. As a result it is no longer needed in the cloud formation template for FreeIPA. In order for the template to process I had to avoid sending the AZ parameter into the template processor. But I needed to only do this for FreeIPA and not core. As a result, I changed FreeIPA to not set the AZ since it is not used that way the common code could check if it was null and optionally use the AZ parameter.